src/main/groovy/com/smartbear/swagger/Swagger2Importer.groovy

```diff
@@ -311,9 +314,14 @@ class Swagger2Importer implements SwaggerImporter {
             }
         }
 
-        RestService restService = forRefactoring ?
-                InterfaceFactoryRegistry.createNew(project, RestServiceFactory.REST_TYPE, name) :
-                project.addNewInterface(name, RestServiceFactory.REST_TYPE)
+        RestService restService;
+        if (forRefactoring) {
```

Comment:
I don't think there is a need to handle refactoring differently - under the hood project.addNewInterface calls InterfaceFactoryRegistry.createNew...